lib/Runtime/Library/JavascriptString.cpp

```diff
@@ -1548,7 +1548,7 @@ namespace Js
         }
 
         // We aren't going to bail early so let's create a StringBuilder and put the first raw string in there
-        CompoundString::Builder<64 * sizeof(void *) / sizeof(char16)> stringBuilder(scriptContext);
+        CompoundString::Builder<64 * sizeof(void *) / sizeof(char16)> stringBuilder = (scriptContext);
```

Comment:
Why change the clear invocation of a constructor to assignment syntax for constructing the object?